pypeit/spectrographs/gemini_gnirs.py

```diff
@@ -304,6 +308,9 @@ def order_spat_pos(self):
         if '10/mmLBSX' in self.dispname:
             return np.array([0.050, 0.215, 0.442, 0.759])
         elif '32/mm' in self.dispname:
```

Comment:
Let's add a TODO here, that we should create self.date similar to self.dispname and use that to decide which numbers to use.

Reply:
add a todo